about.html

```diff
@@ -126,23 +126,7 @@ <h2 class="about-section__main-title">See the best rated games ever</h2>
 
   <footer class="footer-section">
     <div class="footer-section__container">
```

Comment:
Great design @EvansSnave 🤝

- [ ] As stated in the project requirements, kindly ensure your design replicates the given design as much as possible. Kindly add partners to the mobile design of the About section.
- Your design
![Your mobile-about](https://github.com/EvansSnave/capstone-project/assets/63917725/ed435069-f64e-4f71-98c0-393068337563)
- Given design
![Given-mobile-about](https://github.com/EvansSnave/capstone-project/assets/63917725/4db12dce-a7fa-49b2-bd48-03fa44d274f5)